test/e2e/predictor/test_onnx.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2019 kubeflow.org.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import numpy as np
+from kubernetes import client
+
+from kfserving import KFServingClient
+from kfserving import constants
+from kfserving import V1alpha2EndpointSpec
+from kfserving import V1alpha2PredictorSpec
+from kfserving import V1alpha2ONNXSpec
+from kfserving import V1alpha2InferenceServiceSpec
+from kfserving import V1alpha2InferenceService
+from kubernetes.client import V1ResourceRequirements
+
+from ..common.utils import predict
+from ..common.utils import KFSERVING_TEST_NAMESPACE
+
+from PIL import Image
+
+api_version = constants.KFSERVING_GROUP + '/' + constants.KFSERVING_VERSION
+KFServing = KFServingClient(config_file="~/.kube/config")
+ONNX_EXAMPLE_IMAGE_FILE = './data/onnx_input.jpg'
+ONNX_EXAMPLE_JSON_FILE = './data/onnx_input.json'
+
+def prepare_onnx_example(path):
+    im = Image.open(path)
+    return np.array(im.resize((300, 400))).astype('float32') / 255
```

Comment:
I will have to play around with this to make sure this is the shape the mosaic model is expecting.